browser/containers/Admin.tsx

```diff
@@ -100,6 +146,13 @@ class Admin extends Component<Props, State> {
     });
   };
 
+  getProjects = async () => {
+    const projectList = await reqJSON('/api/strategic/projects');
+    this.setState({
```

Comment:
see other comment re setstate; this may not matter here

Reply:
Same as the previous comment.